cmake/modules/SearchRootCoreDeps.cmake

```diff
@@ -10,35 +10,32 @@ message(STATUS "Looking for Python")
 
 if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.14)
 
-  if(PYTHON_EXECUTABLE OR Python_EXECUTABLE)
-    # - Check if the PYTHON_EXECUTABLE deprecated variable or Python_EXECUTABLE
+  if(PYTHON_EXECUTABLE)
```

Comment:
So we don't offer the option to the user to specify `Python_EXECUTABLE` anymore?

Reply:
It's open to discussion, I have no strong opinion honestly. I added it in the last PR because it was used internally, now it's not used anymore and for backward compatibility the only one needed is ```PYTHON_EXECUTABLE```.

Reply:
Ok, for me let's just use PYTHON_EXECUTABLE.